snmp/datadog_checks/snmp/data/default_profiles/dlink-dgs-switch.yaml

```diff
@@ -0,0 +1,169 @@
+extends:
+  - dlink.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.171.10.137.*
+  - 1.3.6.1.4.1.171.10.137.1.1
+
+metrics:
+  - MIB: DLINKSW-ENTITY-EXT-MIB
+    table:
+      name: dEntityExtCpuUtilTable
+      OID: 1.3.6.1.4.1.171.14.5.1.7
+    symbols:
+      - name: cpu.usage
+        OID: 1.3.6.1.4.1.171.14.5.1.7.1.3 # dEntityExtCpuUtilOneMinute
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.171.14.5.1.7.1.1
+          name: dEntityExtCpuUtilUnitId
+        tag: dlink_entity_ext_cpu_util_unit_id
+      - column:
+          OID: 1.3.6.1.4.1.171.14.5.1.7.1.2
+          name: dEntityExtCpuUtilCpuID
+        tag: dlink_entity_ext_cpu_util_cpu_id
+  - MIB: DLINKSW-ENTITY-EXT-MIB
+    table:
+      name: dEntityExtMemoryUtilTable
+      OID: 1.3.6.1.4.1.171.14.5.1.4
+    symbols:
+      - name: memory.total
+        OID: 1.3.6.1.4.1.171.14.5.1.4.1.3
+      - name: memory.used
+        OID: 1.3.6.1.4.1.171.14.5.1.4.1.4
+    metric_tags:
```

Comment:
Same comment as for `cpu.usage` metric tag, we usually tag table memory.* metrics with `mem`